<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notepad</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPlainTextEdit" name="yaziEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>781</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushKaydet">
       <property name="text">
        <string>Kaydet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushTemizle">
       <property name="text">
        <string>Temizle</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuA">
    <property name="title">
     <string>Menü</string>
    </property>
    <addaction name="aksiyonAc"/>
    <addaction name="aksiyonKaydet"/>
    <addaction name="aksiyonFarkliKaydet"/>
   </widget>
   <addaction name="menuA"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="aksiyonAc">
   <property name="text">
    <string>Aç</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="aksiyonKaydet">
   <property name="text">
    <string>Kaydet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="aksiyonFarkliKaydet">
   <property name="text">
    <string>Farklı Kaydet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
